Makefile

```diff
@@ -179,7 +179,8 @@ docs.tar.gz: hail/build/www
 
 website-image: docs.tar.gz
 
-$(SERVICES_IMAGES): %-image: $(SERVICES_IMAGE_DEPS) $(shell git ls-files $$* ':!:**/deployment.yaml')
+.SECONDEXPANSION:
```

Comment:
Sorry this accidentally snuck in here. This line was broken and prevented me from making the batch image locally. I'm happy to put this into a separate PR but the explanation is:

The intention of this line is for `$$*` to refer to the `%` in the target pattern, i.e. if you're building `batch-image` depend on all the tracked files in `batch` (excluding deployment.yaml which the image can never depend on). But with the line that's currently in `main`, the shell invocation happens *before* the target is fully resolved, and you get `git ls-files` which just lists everything in the repo. What [`.SECONDEXPANSION`](https://www.gnu.org/software/make/manual/html_node/Secondary-Expansion.html) does is allows us to defer the evaluation of the shell line so that it does the expected thing when it eventually is evaluated.

Reply:
Sounds good! I'm fine with that going in with the rest of these changes :)